vrk-kpa/suomidigi
# SUOPA-840 Sidebar

To test: 
- `make fresh`
- Open https://suomidigi.docker.sh/ohjeet-ja-tuki/kokonaisarkkitehtuuri
- Open proto https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/c6322fd9-eff8-4da6-90bd-a2642cbf0a0e/Suomidigi-2020-suunnann-ytt-j-t-2019-XL
- Check that the sidebar is similar to proto and that there is Updated date in Kokonaisarkkitehtuuri -(basic) page
